### ISSUE TITLE
Request is not properly formatted when excludeCredentials is set

### ISSUE THREAD

**Issue by arokettu** [namaggarwal/webauth-helpers#5]
The error:
```
TypeError: CredentialsContainer.create: 'id' member of PublicKeyCredentialDescriptor could not be converted to any of: ArrayBufferView, ArrayBuffer.
```

Fix: `excludeCredentials[...].id` should also be converted to buffers

**Comment by namaggarwal** [comment]
Thank you for the issue. This is a valid issue. Marking as Bug

**Comment by namaggarwal** [comment]
This is fixed in the latest version - 0.1.0

**Comment by arokettu** [comment]
thank you
